code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -129,6 +128,7 @@ GLOBAL_LIST_INIT(common_loot, list( //common: basic items
 		/obj/item/reagent_containers/glass/beaker = 1,
 		/obj/item/reagent_containers/glass/rag = 1,
 		/obj/item/reagent_containers/hypospray/medipen/pumpup = 2,
+		/obj/item/reagent_containers/food/drinks/soda_cans/oceanwave = 1,
```

Comment:
And when adding stuff, please try to add ADDITION comments to it if its in the master files
```suggestion
		/obj/item/reagent_containers/food/drinks/soda_cans/oceanwave = 1, // LOBOTOMYCORPORATION ADDITION
```